src/presentational/SuccessDialog.tsx

```diff
@@ -1,17 +1,18 @@
 import React from 'react';
 import {StyleSheet, StyleProp, TextStyle, ViewStyle} from 'react-native';
-import {Layout, Text, Icon} from '@ui-kitten/components';
-import {standardPadding, monofontFamily, colorGreen} from 'src/styles';
+import {Layout, Text, Icon, Button, Divider} from '@ui-kitten/components';
+import globalStyles, {standardPadding, monofontFamily, colorGreen} from 'src/styles';
 
 type PropTypes = {
   text: string;
   inline?: boolean;
   textStyles?: StyleProp<TextStyle>;
   containerStyle?: StyleProp<ViewStyle>;
+  onClosePress?: () => void;
 };
 
 function SuccessDialog(props: PropTypes) {
-  const {text, inline = false, textStyles = {}, containerStyle = {}} = props;
+  const {text, inline = false, textStyles = {}, containerStyle = {}, onClosePress = undefined} = props;
```

Comment:
```suggestion
  const {text, inline = false, textStyles = {}, containerStyle = {}, onClosePress} = props;
```
I think the default value of undefined would be the same as not putting any.